views/js/qtiCreator/widgets/interactions/selectPointInteraction/states/Sleep.js

```diff
@@ -17,7 +38,9 @@ define([
         });
     };
 
-    var exitSleepState = function exitSleepState(){};
+    var exitSleepState = function exitSleepState(){
+        $('.image-editor.solid, .block-listing.source', this.widget.$container).css('min-width', 0);
+    };
```

Comment:
_Note:_ This line is part of https://github.com/oat-sa/extension-tao-itemqti/pull/241